From 7b47a2ff4885fde4e3956d1248603818bfe465b3 Mon Sep 17 00:00:00 2001
From: Dmitry Bogatov <KAction@gnu.org>
Date: Tue, 12 Mar 2019 19:07:03 +0000
Subject: [PATCH] Fix factual error in manpage.

Forwarded: no
Last-Update: 2016-08-31

Log ./finish script is not actually invoked. It is not clear, whether this
limitation is worth lifting.
Last-Update: 2016-08-31
---
 man/runsv.8 | 6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

diff --git a/man/runsv.8 b/man/runsv.8
index 53f1b1b..348bf3c 100644
--- a/man/runsv.8
+++ b/man/runsv.8
@@ -54,10 +54,8 @@ and
 .IR service /finish's
 standard output to the pipe, switches to the directory
 .IR service /log
-and starts ./run (and ./finish) exactly as described above for the
-.I service
-directory.
-The standard input of the log service is redirected to read from the pipe.
+and starts ./run script. The standard input of the log service
+is redirected to read from the pipe.
 .P
 .B runsv
 maintains status information in a binary format (compatible to the
-- 
2.30.0

